CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,8 @@
 ### SDK Features
+* `service/s3/s3manager`: Add Download Buffer Provider [(#2823)](https://github.com/aws/aws-sdk-go/pull/2823)
```

Comment:
Should also include a reference to fixing https://github.com/aws/aws-sdk-go/issues/2180

Reply:
The formating of the PR GitHub link has a typo in the title. it should be the following with the square brackets `[]` surrounding the link.
```
[(#2823)(https://github.com/aws/aws-sdk-go/pull/2823)]
```